packages/expo-media-library/src/MediaLibrary.ts

```diff
@@ -430,3 +430,20 @@ export async function getMomentsAsync() {
 
   return await MediaLibrary.getMomentsAsync();
 }
+
+// Android only
+export async function migrateAlbumAsync(album: AlbumRef): Promise<void> {
```

Comment:
I think we should clarify it's not always migrating, but only when it's needed 😉 Could you also add tsdocs for these new methods?

```suggestion
export async function migrateAlbumIfNeededAsync(album: AlbumRef): Promise<void> {
```